acceptance/tests/invalid_ssl_config.rb

```diff
@@ -36,7 +36,8 @@
     create_remote_file(agent, pxp_agent_config_file(agent), pxp_config_json_using_puppet_certs(master, agent).to_s)
     on agent, puppet('resource service pxp-agent ensure=running')
 
-    assert(is_associated?(master, "pcp://#{agent}/agent"),
+    inventory_retries = 60
+    assert(is_associated?(master, "pcp://#{agent}/agent", inventory_retries),
```

Comment:
It looks like the default is already 60: https://github.com/puppetlabs/pxp-agent/blob/274b0265cbeb3824df1fbb96d9b35e34469dfdf1/acceptance/lib/pxp-agent/test_helper.rb#L204. Makes me think this test is subject to race conditions...